pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -234,3 +254,84 @@ format_annot <- function(annot_list) {
   rownames(annot) <- annot$input
   return(annot)
 }
+
+# Fix annotations makes annotations compatible between samples with different types.
```

Comment:
Fix annotations does not seem to be the name of the function anymore; also `equalize` sounds a bit weird, maybe it's me but the name did not make it clear what the function does.

Reply:
Yeah also sounded weird to me, but it's trying to make all the annotations equal. I'll think alternatives

Reply:
what about normalise or standardise?

Reply:
or normalize/standardize if you'd rather go _'murica style_